test/routes/route_test.rb

```diff
@@ -0,0 +1,24 @@
+require "test_helper"
+
+class RouteTest < ActionDispatch::IntegrationTest
+  test "root" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.